source/frontend/src/features/mapSideBar/lease/common/LeaseHeader.tsx

```diff
@@ -43,19 +43,19 @@ export const LeaseHeader: React.FC<ILeaseHeaderProps> = ({ lease, lastUpdatedBy
   return (
     <Container>
       <Row className="no-gutters">
-        <StyledLeftHeaderPane xs="8">
-          <HeaderField label="Lease/Licence #" labelWidth="4" contentWidth="8">
+        <StyledLeftHeaderPane xl="8" xs="12">
```

Comment:
migrate the "split left & right" content to all display on their own rows for small screens.